### ISSUE TITLE
Attaching Units to calls in MDT not working as intended?

### ISSUE THREAD

**Issue by Spudgun** [Project-Sloth/ps-mdt#447]
Tried to attach units to a call and it doesn't work via the MDT. I believe it is because of the code below ([Link Here](https://github.com/Project-Sloth/ps-mdt/blob/dd52e59dbbd82f273c3366ba06816c97baf71250/server/main.lua#L1548))

```lua
RegisterNetEvent('mdt:server:callDragAttach', function(callid, cid)
	local src = source
	local Player = QBCore.Functions.GetPlayer(src)
	local playerdata = {
		name = Player.PlayerData.charinfo.firstname.. " "..Player.PlayerData.charinfo.lastname,
		job = Player.PlayerData.job.name,
		cid = Player.PlayerData.citizenid,
		callsign = Player.PlayerData.metadata.callsign
	}
	local callid = tonumber(callid)
	local JobType = GetJobType(Player.PlayerData.job.name)
	if JobType == 'police' or JobType == 'ambulance' then
		if callid then
			TriggerEvent('dispatch:addUnit', callid, playerdata, function(newNum)
				TriggerClientEvent('mdt:client:callAttach', -1, callid, newNum)
			end)
		end
	end
end)
```

The event `'dispatch:addUnit'` does not point to anything in the MDT or ps-dispatch. Is this intended? I thought functionality would be along the lines of attaching an officer to a call, it would then send them a notification and update their GPS to the location of the call via ps-dispatch. Essentially adding functionality to the MDT for a dispatcher type role to manage the police force.

**Comment by MonkeyWhisper** [comment]
You're using outdated ps-mdt.

**Comment by Spudgun** [comment]
This is from the most recently released version - 2.6.6 found [here](https://github.com/Project-Sloth/ps-mdt/releases/tag/2.6.6)
